pages/price-feeds/getting-started.mdx

```diff
@@ -0,0 +1,15 @@
+## Getting Started
+
+Integrating Pyth price feeds into your application is quick and easy.
+
+### Build 🧑‍💻
+
+- Jump to our quick [tutorial](./create-your-first-pyth-app/) to get started with Pyth price feeds in your application.
+- Consult our [guides](./use-real-time-data/) and selected the appropriate chain for your application.
```

Comment:
Please try not to use "our" in the documentation -- who is "we" in this case? Pyth is a protocol not a company or team. 

In this case "the" would be a good fix

Reply:
I also think these bullets should tell the reader which of the resources they should go to. The tutorial is designed for new developers, whereas the how to guide is for more experienced builders who largely know what they're doing.

Reply:
I also think the titles of the linked documents should be in the text so that the reader can map the linked docs to the items in the nav menu on the left.